### ISSUE TITLE
Conflict with R8 of AGP 7.4.0-alpha09

### ISSUE THREAD

**Issue by qdsfdhvh** [JetBrains/markdown#106]
When I upgarded AGP to `7.4.0-alpha09` and run `./gradlew :android:minifyReleaseWithR8`,  I get the following error:
```
Execution failed for task ':app:android:minifyReleaseWithR8'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.R8Task$R8Runnable
   > Compilation failed to complete, position: Lorg/intellij/markdown/parser/sequentialparsers/impl/ReferenceLinkParser$Companion;parseShortReferenceLink(Lorg/intellij/markdown/parser/sequentialparsers/TokensCache$Iterator;)Lorg/intellij/markdown/parser/sequentialparsers/LocalParsingResult;, origin: /Users/seiko/.gradle/caches/modules-2/files-2.1/org.jetbrains/markdown-jvm/0.3.1/45084b8ac961f3c5247a2289058c7522af157876/markdown-jvm-0.3.1.jar:org/intellij/markdown/parser/sequentialparsers/impl/ReferenceLinkParser$Companion.class
```

**Comment by YektaDev** [comment]
What happened?

**Comment by qdsfdhvh** [comment]
google has fixed this issue with the upcoming AGP 7.4 beta-1,  https://issuetracker.google.com/issues/241636314
